### PR TITLE
keep feature

### PR DESCRIPTION
**Pull request by kibotu**
See #76

### CODE REVIEW

**Comment by x2on** [comment]
Thanks for your PR. 
Please revert code style changes.
Then i can merge it.

**Comment by kibotu** [comment]
hi, thanks for your consideration, i also fixed #85

regarding the code style changes, which ones do you mean exactly? can you highlight them with a tiny comment please? 

thanks, best regards
jan

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,131 @@
-*.ipr
+
+# Created by https://www.gitignore.io/api/jetbrains,java,osx,windows,gradle
+
+### JetBrains ###
```
Comment (x2on) [inline .gitignore]: Please remove this change

File: src/main/groovy/de/felixschulze/gradle/HockeyAppPlugin.groovy [hunk 2]
```diff
@@ -24,12 +24,12 @@
 
 package de.felixschulze.gradle
 
+import com.android.build.gradle.AppExtension
+import com.android.build.gradle.AppPlugin
 import com.android.build.gradle.api.ApplicationVariant
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.gradle.api.Task
```
Comment (x2on) [inline src/main/groovy/de/felixschulze/gradle/HockeyAppPlugin.groovy]: Please revert

File: src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy [hunk 3]
```diff
@@ -93,8 +93,7 @@ class HockeyAppUploadTask extends DefaultTask {
                 logger.debug('Mapping file not found')
                 mappingFileCouldBePresent = false
             }
-        }
-        else {
+        } else {
```
Comment (x2on) [inline src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy]: Please revert code style change

File: src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy [hunk 4]
```diff
@@ -151,9 +150,10 @@ class HockeyAppUploadTask extends DefaultTask {
 
     }
 
-    def void uploadFilesToHockeyApp(File appFile, @Nullable File mappingFile, @Nullable String appId) {
+    def void uploadFilesToHockeyApp(File appFile,
```
Comment (x2on) [inline src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy]: Please revert code style change

File: src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy [hunk 5]
```diff
@@ -193,15 +193,14 @@ class HockeyAppUploadTask extends DefaultTask {
...
-
         int lastProgress = 0
         Logger loggerForCallback = logger
         Boolean teamCityLog = hockeyApp.teamCityLog
         ProgressHttpEntityWrapper.ProgressCallback progressCallback = new ProgressHttpEntityWrapper.ProgressCallback() {
 
             @Override
             public void progress(float progress) {
-                int progressInt = (int)progress
+                int progressInt = (int) progress
```
Comment (x2on) [inline src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy]: Please revert code style change

File: src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy [hunk 6]
```diff
@@ -226,8 +225,8 @@ class HockeyAppUploadTask extends DefaultTask {
 
         if (response.getStatusLine().getStatusCode() != HttpStatus.SC_CREATED) {
             parseResponseAndThrowError(response)
-        }
-        else {
+        } else {
```
Comment (x2on) [inline src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy]: Please revert code style change

**Comment by x2on** [comment]
Please pull the current master into your branch

**Comment by x2on** [comment]
Should purge getting default?

**Comment by kibotu** [comment]
thanks, going to have a look as soon as i can

**Comment by x2on** [comment]
Any news?

**Comment by kibotu** [comment]
crap totally forgot about it, will have a look this week D: sorry ^^'

**Comment by x2on** [comment]
Any news?

**Comment by x2on** [comment]
I close this during inactivity. Feel free to open again.

**Pull request closed without merging**